src/generic_core/uproxy_core.ts

```diff
@@ -368,8 +368,12 @@ export class uProxyCore implements uproxy_core_api.CoreApi {
       // Assumes the user is only signed in once to any given network.
       networkUserId = Object.keys(social_network.networks[networkName])[0];
     }
-    var network = social_network.getNetwork(networkName, networkUserId);
-    return network.acceptInvitation(data.tokenObj, data.userId);
+    try {
```

Comment:
That's true inside a Promise context - but if the caller is not itself inside a Promise context, weird things will happen. This is one reason I'm a little down on Promises :-)